doc/tutorial/Makefile

```diff
@@ -1,8 +1,4 @@
 notebooks:
-
-	ipython nbconvert --to rst ../../../scikit-beam-examples/demos/1_time_correlation/*.ipynb
-	ipython nbconvert --to rst ../../../scikit-beam-examples/demos/dpc/*.ipynb
-	ipython nbconvert --to rst ../../../scikit-beam-examples/demos/reciprocal_space/*.ipynb
-	ipython nbconvert --to rst ../../../scikit-beam-examples/demos/speckle/*.ipynb
-	ipython nbconvert --to rst ../../../scikit-beam-examples/demos/xrf/*.ipynb
-	ipython nbconvert --to rst ../../../scikit-beam-examples/demos/*.ipynb
+	git clone https://github.com/scikit-beam/scikit-beam-examples /tmp/scikit-beam-examples
+	python write_main_index.py
+	ipython nbconvert --config convert_config.py --to rst
```

Comment:
interesting. so this line will convert all the ipython notebook according to configuration file convert_config.py.